powershell/resources/psruntime/BuildTime/Cmdlets/ExportProxyCmdlet.cs

```diff
@@ -68,9 +68,9 @@ protected override void ProcessRecord()
                         .Select(pvg => new ProfileGroup(pvg.Select(pv => pv.variant).ToArray(), pvg.Key))
                     : new[] { new ProfileGroup(variants) };
                 var variantGroups = profileGroups.SelectMany(pg => pg.Variants
-                    .GroupBy(v => new { v.CmdletName, v.IsInternal })
+                    .GroupBy(v => new { v.CmdletName, v.IsInternal, v.IsModelCmdlet })
```

Comment:
We should use the following strategy to determine if a variant group is IsModelCmdlet:
All variants within the group must be IsModelCmdlet.